src/status_im/ui/screens/wallet/send/views.cljs

```diff
@@ -70,40 +67,56 @@
 
 (defn render-contact [contact from-chat?]
   (if from-chat?
-    [list-item/list-item {:title (multiaccounts/displayed-name contact)
+    [list-item/list-item {:title    (multiaccounts/displayed-name contact)
                           :subtitle (:address contact)
-                          :icon  (multiaccounts/displayed-photo contact)}]
+                          :icon     (multiaccounts/displayed-photo contact)}]
     [list-item/list-item
-     {:title       (utils/get-shortened-checksum-address
-                    (if (string? contact) contact (:address contact)))
-      :subtitle    (when-not contact (i18n/label :t/wallet-choose-recipient))
+     {:title               (utils/get-shortened-checksum-address
+                            (if (string? contact) contact (:address contact)))
+      :subtitle            (when-not contact (i18n/label :t/wallet-choose-recipient))
       :accessibility-label :choose-recipient-button
-      :on-press    #(do
-                      (re-frame/dispatch [:dismiss-keyboard])
-                      (re-frame/dispatch [:bottom-sheet/show-sheet
-                                          {:content        sheets/choose-recipient
-                                           :content-height 200}]))
-      :accessories [:chevron]}]))
+      :on-press            #(do
+                              (re-frame/dispatch [:dismiss-keyboard])
+                              (re-frame/dispatch [:bottom-sheet/show-sheet
+                                                  {:content        sheets/choose-recipient
+                                                   :content-height 200}]))
+      :accessories         [:chevron]}]))
+
+(defn set-max [token]
+  [react/touchable-highlight
+   {:on-press
+    #(when token
+       (re-frame/dispatch [:wallet.send/set-max-amount token]))}
+   [react/view {:height            35 :border-radius 40 :background-color colors/blue-light
```

Comment:
This looks unreadable to me, do you mind to add comas here or keep one pair per line?

Reply:
done